src/govpress-unit/ways-of-working.md

```diff
@@ -0,0 +1,78 @@
+We are not dogmatic about exactly who works on which piece of work, but this document describes our expected way of working throughout the year.
+
+## Sprinting teams
+
+Sprinting teams work on longer (>2 weeks) projects which clients pay for separately (i.e. not via retainers). Sprinting teams are not static, we create them when we have work available and each team is chosen to meet the needs of the work.
+
+Note that sprinting teams will often be small. Where architecture and code review become a burden, the team may use pair or mob programming to speed up delivery.
+
+Team members will be scheduled for work on this team, before each project begins. These schedules will be coordinated by the Delivery Lead.
+
+### The team
+
+This team:
+
+* Is led by a Tech Lead (usually the Projects Lead Developer) and Delivery Lead
+* Is formed around the needs of each project, but each developer should work in a sprinting team over the course of a year
+* Usually includes one front-end developer or designer
+* Only includes the GovPress Ops Engineer and Platform Tech Lead when necessary
+* Uses the `#dxw-govpress-sprinting-teams` Slack channel for coordination
+
+### Cadence
+
+Each project in this team will run roughly like a Scrum project:
+
+* Some discovery work is done before the project starts and led by the Delivery Lead or Head of Unit
+* Projects start with one (or more) days of technical preparation time with only the Tech Lead working on the project, while other team members help on Support
+* Iterations happen in 2 week sprints
+* Ceremonies are run by the Delivery Lead and will include both client-facing stand-ups and time for the team to synchronise without the client
+* The Delivery Lead checks in with the client in the middle of sprints (outside of stand-ups)
+* Projects usually include a retrospective with client at the end of the project
+* Team members have 1 dxw day at end of each project before they start the next project
+
+## Maintenance and support team
+
+This team works on tasks which are paid for via support and maintenance retainers. Some members of the team may work on tasks that are paid for separately, but last much less than a 2 week Scrum sprint.
+
+Where we have larger pieces of work that require several developers or longer periods of time (including site migrations), we create a new sprinting team, led by one of the Tech Leads.
+
+Where we have larger pieces of work (>1day) that are too small for a sprinting team, these are taken on by a minimum of two developers, one of which will usually be a Tech Lead.
+
+The team takes overall responsibility for the maintenance of our products and internal tooling. However, everyone is encouraged and expected to resolve maintenance issues when they come across them, or raise Trello cards for this team where tasks are too large to be completed in the course of other work.
```

Comment:
Suggestion: 

`...for the development and maintenance of our products and internal tooling...`

Hopefully we're creating new things as well as stopping the existing things breaking!

Reply:
Thanks, pushed a change for that